docs/howto/outside_controller_use.md

```diff
@@ -0,0 +1,37 @@
+## Using AMS Outside Of A Controller
+
+### Serializing a resource 
+
+In the current version of AMS, serializing resources outside controllers is
+fairly simple:
+
+```ruby
+# Create our resource
+post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::SerializableResource.serialize(post, options)
+
+# Create a json representation on your serialized model
+model_json = serializer.as_json
+```
+
+### Retrieving a Resource's Active Model Serializer
+
+If you want to retrieve a serializer for a specific resource, you can do the following:
+
+```ruby
+# Creating the resource
+post = Post.create(title: "Another Example", body: "So much fun.")
+
+# Optional options parameters
+options = {}
+
+# Create the serializer
+serializer = ActiveModel::Serializer.serializer_for(post, options)
```

Comment:
Just to clarify, this is the serializer class i.e. `serializer == PostSerializer`, not the serializer instance (same below).

ref: https://github.com/rails-api/active_model_serializers/blob/64168cbecd6dfff99d81e4ab5444b36ffe219beb/lib/active_model/serializable_resource.rb#L46-L65